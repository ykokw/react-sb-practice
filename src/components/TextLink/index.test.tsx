import React from "react";
import Renderer from "react-test-renderer";

import {
  Primary,
  Secondary,
  PrimaryBold,
  SecondaryBold,
} from "./index.stories";

test("Primary", () => {
  const component = Renderer.create(<Primary />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("Secondary", () => {
  const component = Renderer.create(<Secondary />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("PrimaryBold", () => {
  const component = Renderer.create(<PrimaryBold />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("SecondaryBold", () => {
  const component = Renderer.create(<SecondaryBold />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
