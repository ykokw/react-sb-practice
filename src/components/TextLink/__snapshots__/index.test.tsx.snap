// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Primary 1`] = `
.c0 {
  color: #61dafb;
  font-weight: normal;
}

<a
  className="c0"
  color="blue"
  href="http://localhost:6006"
>
  TextLink
</a>
`;

exports[`PrimaryBold 1`] = `
.c0 {
  color: #61dafb;
  font-weight: bold;
}

<a
  className="c0"
  color="blue"
  href="http://localhost:6006"
>
  TextLink
</a>
`;

exports[`Secondary 1`] = `
.c0 {
  color: #ccc;
  font-weight: normal;
}

<a
  className="c0"
  color="gray"
  href="http://localhost:6006"
>
  TextLink
</a>
`;

exports[`SecondaryBold 1`] = `
.c0 {
  color: #ccc;
  font-weight: bold;
}

<a
  className="c0"
  color="gray"
  href="http://localhost:6006"
>
  TextLink
</a>
`;
